packages/blitz/src/cli/commands/generate.ts

```diff
@@ -227,9 +271,20 @@ const generate: CliCommand = async () => {
 
     const generators = generatorMap[selectedType as keyof typeof generatorMap]
 
+    const isTypeScript = await getIsTypeScript()
+    const blitzServerPath = isTypeScript ? "app/blitz-server.ts" : "app/blitz-server.js"
+    const blitzServer = require("path").join(process.cwd(), blitzServerPath)
+    const {register} = require("esbuild-register/dist/node")
+    const {unregister} = register({
+      target: "es6",
+    })
+    const blitzConfig = require(blitzServer).default
```

Comment:
I think we should do this differently. Maybe not export the config object as default and instead add as a property to `setupBlitzServer`.

This may be a design choice though. @flybayer @beerose  what do you think?

Reply:
I suggested this solution as this is what we agreed on during a planning call last week @dillonraphael.

Reply:
We agreed to export just the config, not as a property of setupBlitzServer, however I think we should use a named export of `cliConfig` instead of a default export